src/web_interface/templates/compare/text_files.html

```diff
@@ -21,7 +21,7 @@
 <script>
 document.addEventListener('DOMContentLoaded', () => {
     // This string is made safe by the python code rendering this template
```

Comment:
This comment should be removed.

Reply:
Removed. That is the problem with having many comments: they also need maintenance :sweat_smile: